src/metal.cs

```diff
@@ -180,6 +185,21 @@ partial interface MTLBuffer : MTLResource {
 #endif
 		[Export ("removeAllDebugMarkers")]
 		void RemoveAllDebugMarkers ();
+
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[NoiOS, NoTV, Mac (10, 15)]
+		[NullAllowed, Export ("remoteStorageBuffer")]
+		IMTLBuffer RemoteStorageBuffer { get; }
+
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[NoiOS, NoTV, Mac (10,15)]
+		[Export ("newRemoteBufferViewForDevice:")]
+		[return: NullAllowed]
+		IMTLBuffer Create (IMTLDevice device);
```

Comment:
it's not `static` and there's already other `Create*` methods
better name it `CreateRemoteBuffer` since new API could make the signature conflictual